core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1603,7 +1602,8 @@ public Label getLabel(String expr) {
      * Returns the label atom of the given name.
      * @return non-null iff name is non-null
      */
-    public @Nullable LabelAtom getLabelAtom(@CheckForNull String name) {
+    @CheckForNull
+    public LabelAtom getLabelAtom(@CheckForNull String name) {
```

Comment:
> Name can be passed not from obvious variables, i.e. from method that may return null in some cases.

Well in that case it is up to you, FindBugs cannot help. But code which passes in non-null names (the common case I think) should not be penalized with a meaningless nullness check.

The proper solution is to just deprecated this method and introduce a variant which requires a `@Nonnull String name`.